content/post/twitter-hashtags-real-time.md

```diff
@@ -0,0 +1,561 @@
+                data: {
+                    labels: [{% for item in hashtags_data %}
+                              "{{item}}",
+                             {% endfor %}],
+                    datasets: [{
+                        label: 'Hashtags Counts',
+                        data: [{% for item in counts_data %}
+                                  {{item}},
+                                {% endfor %}],
+                        backgroundColor: [
+                            'rgba(255, 99, 132, 0.2)',
+                            'rgba(54, 162, 235, 0.2)',
+                            'rgba(255, 206, 86, 0.2)',
+                            'rgba(75, 192, 192, 0.2)',
+                            'rgba(153, 102, 255, 0.2)',
+                            'rgba(255, 159, 64, 0.2)',
+                            'rgba(255, 99, 132, 0.2)',
+                            'rgba(54, 162, 235, 0.2)',
+                            'rgba(255, 206, 86, 0.2)',
+                            'rgba(75, 192, 192, 0.2)',
+                            'rgba(153, 102, 255, 0.2)'
+                        ],
+                        borderColor: [
+                            'rgba(255,99,132,1)',
+                            'rgba(54, 162, 235, 1)',
+                            'rgba(255, 206, 86, 1)',
+                            'rgba(75, 192, 192, 1)',
+                            'rgba(153, 102, 255, 1)',
+                            'rgba(255, 159, 64, 1)',
+                            'rgba(255,99,132,1)',
+                            'rgba(54, 162, 235, 1)',
+                            'rgba(255, 206, 86, 1)',
+                            'rgba(75, 192, 192, 1)',
+                            'rgba(153, 102, 255, 1)'
+                        ],
+                        borderWidth: 1
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAxes: [{
+                            ticks: {
+                                beginAtZero:true
+                            }
+                        }]
+                    }
+                }
+           });
+
+
+           var src_Labels = [];
+           var src_Data = [];
+
+            setInterval(function(){
+                $.getJSON('/refreshDashboard', {
+                }, function(data) {
+                    src_Labels = data.r_hashtags;
+                    src_Data = data.r_counts;
+                });
+
+                myChart.data.labels = src_Labels;
+                myChart.data.datasets[0].data = src_Data;
+                myChart.update();
+
+            },1000);
+
+</script>
+```
+
+### 13. Run The Application
+
+Before we run our Twitter Trending Hashtags application, we need to make sure we have the proper Python dependencies installed. We depend on `pandas`, `requests_oauthlib`, and `flask`. These can be installed with the following commands:
+
+```
+pip install pandas requests_oauthlib flask
```

Comment:
This installation takes a long time, and could be moved to an earlier point in the article, so it runs while the user is setting up other things, like the twitter app auth tokens

Reply:
Also this needed to be run with sudo if not run in a virtualenv (on the Ubuntu Trusty VM I was using)